content/blog/2017-03-07-engineering-leads-and-mothers.md

```diff
@@ -0,0 +1,40 @@
+
++++
+
+Two of the biggest changes in my life over the last year in work and personal life were moving from being an engineering more into a management/leadership role and having a kid 5 months ago with my girlfriend.
+
+When reflecting about both changes I could see similarities between how my move into management and our kid impacted me and my girlfriend in different ways. Especially how we perceive our own productivity and what that means for our own personal happiness.
+
+Moving from engineering to management/leadership can be really hard, it definitely was for me. It took me quite a while so it was interesting to see the same patterns come up when my girlfriend started to be a stay-at-home mom.
+
+## My issues when going from engineering to management
+
+While the leadership part of moving into management was a large hurdle and deserves its own post in the future, I want to focus on how I started to perceive my own productivity and how that made me super unhappy for a long time. 
+
+I had a really hard time adjusting to the [manager schedule](http://www.paulgraham.com/makersschedule.html) as it is vastly different from the maker schedule, sitting down and coding for hours.
+
+Over time at Codeship I did a lot of marketing, sales and product work and less and less engineering. For a long time this felt unproductive because often I was just supporting somebody elses goal, like making a specific sale, helping with content or giving feedback on product strategy. While I think (and hope) that helped those specific people to get their stuff done, I didn't feel like I was shipping anything myself. 
+
+So sometimes I would just build something because I wanted to feel like I was shipping. This wasn't particularly productive for me though and frustrated the engineering team.
+
+Now I see similar patterns happening to my girlfriend over the last months. A young kid is obviously a ton of work. Often its not just about doing specific things, but just being there with the kid, holding her when she is sleeping, because she needs a body to sleep on. This means you're just sitting there and can't really do anything directly as it would wake her up. At the end of the day this often felt frustrating to my girlfriend because she didn't feel like she got anything done that day, her list of things to do at home or in the garden grew longer, when in fact she was there for the kid the whole day.
+
+When you're in those situations you need to adjust what you perceive as being productive work. You need to see the unblocking of a colleague and watching TV with your sleeping kid as a task that was also shipped. Of course this should never be an excuse for getting less stuff done than you can or want to. Just that you need to be realistic what impact you can and should have on your team or your family and connecting your own personal happiness to that and not a past measurement of success or impact.
```

Comment:
I like to think about the time that I can help save by managing something, and then how many other things can get done in that time. If I save a developer even 30 minutes in a day, those chunks add up and we can slowly start moving faster and shipping more things. This might not feel personally satisfying at first, but your job as a manager is to magnify and amplify the work of others. Manager help engineers achieve higher goals, accomplish more, and ship more things.